src/kolmogorov_solve.jl

```diff
@@ -0,0 +1,47 @@
+struct NNKolmogorov{C,O} <: NeuralNetDiffEqAlgorithm
+    chain::C
+    opt::O
+   
+end
+NNKolmogorov(chain  ; opt=Flux.ADAM(0.1)) = NNKolmogorov(chain , opt)
+ 
+function DiffEqBase.solve(
+    prob::KolmogorovPDEProblem,
+    alg::NNKolmogorov;
+    abstol = 1f-6,
+    verbose = false,
+    maxiters = 300,
+    save_everystep = false,
+    dt,
+    )
+
+    tspan = prob.tspan
+    phi = prob.phi
+    xspan = prob.xspan
+    sigma = prob.sigma
+    μ = prob.μ
+    d = prob.d
+    T = tspan[2]
+    #hidden layer
+    chain  = alg.chain
+    opt    = alg.opt
+
+    ps     = Flux.params(chain)
+    xi     = rand(Uniform(xspan[1] , xspan[2]), d , maxiters )
+    N = Normal(0 , sqrt(2. *T ))
+    x_sde = xi + μ(xi)*T + sigma(xi)*rand(N , d , maxiters)
```

Comment:
this should instead be defining SDEProblem and using a higher order SDE solver: this is a very inaccurate method.

Reply:
I have changed this to the sde solver.